### PR TITLE
ES2015 Named Export Refactor

### PR DESCRIPTION
**Pull request by ahumphreys87**
* Export all items as named exports so that module loaders such as
  rollup can perform tree shaking.
* Removed unneeded getInstance and moduleHelper.
* Removed numeral in favour of numbro in the currencyHelper.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "backbone": "^1.3.3",
     "backbone-routing": "^0.2.0",
-    "backbone-validation": "^0.11.5",
+    "backbone-validation": "git://github.com/ahumphreys87/backbone.validation.git#master",
```
Comment (mintbridge) [inline package.json]: is this temp?

File: src/helpers/currency.js [hunk 2]
```diff
@@ -31,8 +31,8 @@ class CurrencyHelper {
       formatStr += '0';
     }
 
-    return numeral(amount).format(formatStr);
+    return numbro(amount).format(formatStr);
   }
 }
 
-export default new CurrencyHelper();
+export var Currency = new CurrencyHelper();
```
Comment (mintbridge) [inline src/helpers/currency.js]: var?

Comment (ahumphreys87) [inline src/helpers/currency.js]: whoops

**Comment by mintbridge** [comment]
looks good to me, we should prob do somethings about the validation module longer term though

**Pull request merged**